### PR TITLE
Added etcd configuration for Bertha URLs

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: ansible/userdata/default_instance_user_data.yaml [hunk 1]
```diff
@@ -115,6 +115,8 @@ coreos:
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/synthetic-image-publication-monitor-aws/test-uuid '{{aws_image_monitor_test_uuid}}'       >/dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/synthetic-image-publication-monitor-coco/test-uuid '{{coco_image_monitor_test_uuid}}'     >/dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/kafka-bridge/ucs_queue_proxy_addrs '{{bridging_message_queue_proxy}}'                     >/dev/null 2>&1 || true;"
+		  ExecStart=/bin/sh -c "etcdctl mk /ft/config/bertha/urls/authors '{{authors_bertha_url}}'                                              >/dev/null 2>&1 || true;"
```
Comment (tamas-molnar) [inline ansible/userdata/default_instance_user_data.yaml]: Can you please follow the existing indentation level?

File: provision.sh [hunk 2]
```diff
@@ -23,6 +23,8 @@ AMI=`curl -s https://coreos.com/dist/aws/aws-stable.json | jq -r '.["eu-west-1"]
   bridging_message_queue_proxy=${BRIDGING_MESSAGE_QUEUE_PROXY:=https://kafka-proxy-iw-uk-p-1.glb.ft.com,https://kafka-proxy-iw-uk-p-2.glb.ft.com} \
   varnish_access_credentials=${CLUSTER_BASIC_HTTP_CREDENTIALS} \
   api_host=${API_HOST:=http://api.ft.com} \
+  authors_bertha_url=${AUTHORS_BERTHA_URL} \
```
Comment (tamas-molnar) [inline provision.sh]: Is there a sensible default value we can use in case none is set in etcd?

Comment (izzyblues) [inline provision.sh]: No, there isn't a sensible default value. Actually, this configuration is necessary only for publishing clusters.

File: ansible/userdata/default_instance_user_data.yaml (hunk 1 above)
Comment (izzyblues) [inline ansible/userdata/default_instance_user_data.yaml]: Sorry. My editor fault. Fixed with vi.

File: README.md [hunk 3]
```diff
@@ -64,6 +64,11 @@ export CLUSTER_BASIC_HTTP_CREDENTIALS=
 ##The default is prod DNS name given by Mashery but if we are creating a cluster not for prod then some different DNS name may be required, e.g. http://test.api.ft.com or some such
 export API_HOST=http://api.ft.com
 
+##URLs to Bertha endpoints  for accessing to specific Google Spreadsheet data.
+##AUTHORS_BERTHA_URL refers to the spredsheet of curated authors data.
+##ROLES_BERTHA_URL refers to the spredsheet of roles for curated authors.
```
Comment (tamas-molnar) [inline README.md]: Typo in comments `spredsheet`

Comment (izzyblues) [inline README.md]: Ops... Fixed it.

**Comment by tamas-molnar** [comment]
+1

**Pull request merged**